frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -458,7 +464,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               isValid={!isValidationError(busValidation)}
```

Comment:
btw shouldn't we change TemplateValidations to react to cd-roms in this PR as well. And allow sata's for now until common-templates are fixed?